### DISCUSSION TITLE
Version 10.0.1.0

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#805] [Announcements]
<ul>
        <li>Fixed: Text-to-speech audio amplifier does not allow the PC to enter sleep mode</li>
        <li>Some fixes to the color of the address bar extension icon</li>
        <li>Defined minimum size for the translate button in the Android interface</li>
        <li>Update interface translations</li>
    </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v10.0.1.0'>Version 10.0.1.0</a>.</em>

**Comment by aifanghh** [comment]
怎么加入安卓浏览器
